business/istio_status.go

```diff
@@ -96,8 +108,13 @@ func (iss *IstioStatusService) getIstioComponentStatus(ctx context.Context, clus
 				IsCore:    true,
 			})
 		}
+		if cp.ManagesExternal {
+			managesExternal = true
+		}
 	}
-	if len(istiodStatus) == 0 {
+
+	// if no control plane and no any other control plane which manages external
+	if len(istiodStatus) == 0 && !managesExternal {
```

Comment:
I don't think this covers the case where a controlplane may be managing an external cluster but it's not the current cluster. e.g.

You have 3 clusters: A, B, C. A manages B but nothing manages C.

I don't think you need to look at `ManagesExternal`. You just need to know if there's a controlplane managing _this_ cluster or not e.g. `if cluster not in controlPlanes.managedClusters`.

Reply:
I'm also not sure if this is always an error. You could have an "observability" cluster for instance where the observability components live separate from any istio components.

Reply:
A problem with showing an error when the istiod is not found is that a user may not have access to the cluster where the istiod lives so this would show an error even though the istiod does exist.

Reply:
> do we have somewhere documented all those combinations? or it is only here? https://istio.io/latest/docs/setup/install/multicluster/

It's just there but there but you can also combine some of those e.g. external control plane + primary-remote.